src/utils.js

```diff
@@ -59,7 +60,7 @@ async function getConfig(file: string): Promise<ZelConfig> {
  * @param {string} file
  * @return {Promise<T>}
  */
-async function sync(repo: string, branch: string, file: string): Promise<T> {
+async function sync<T>(repo: string, branch: string, file: string): Promise<T> {
```

Comment:
> Lol, not being able to use a library because of typings doesn't bode well for the pro-typings camp 😅

Agreed though `bluebird` is just a ponyfill to native Promises themselves with added sugar. Also I may replace Caporal with an alternative to be more minimal or not even require a CLI framework since we're pretty `zel` itself is really basic as is.